sdk/storage/Azure.Storage.DataMovement/src/ServiceToServiceJobPart.cs

```diff
@@ -177,8 +177,10 @@ public static ServiceToServiceJobPart CreateJobPartFromCheckpoint(
                 createPreference: createPreference);
         }
 
-        public override async Task ProcessPartToChunkAsync()
+        public override async Task ProcessPartToChunkAsync(
+            CancellationToken cancellationToken = default)
         {
+            _cancellationToken = CancellationTokenSource.CreateLinkedTokenSource(_cancellationToken, cancellationToken).Token;
```

Comment:
I think we want to assign this to the local parameter, not the instance variable.

Reply:
Please refer to [sdk/storage/Azure.Storage.DataMovement/src/JobPartInternal.cs](https://github.com/Azure/azure-sdk-for-net/pull/46712/files/a827679371aca391edb76937637634b83c675a43#diff-404e5da065ff87a6ec796d0e1cfc16538e3977dbdfba38ca67252c09c7c39a3f)